notebooks/21_kaggle_data_cleaning.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p>I am sorry that I was confused about the cleaned example.</p><p>Is this expected to remove <code>I, but I, doing it, etc.. </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/datumaro/pull/1534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
This [link](https://gist.github.com/sebleier/554280) is the list of stopwords, and in our clean transformation we remove html tags, urls, emoticons, special characters and punctuation. So in this case the result is removed stopwords and special characters. If you think this index is not appropriate to the example, I will change the number of index.